docs/interpreter/livy.md

```diff
@@ -105,3 +125,6 @@ The session would have timed out, you may need to restart the interpreter.
 > Blacklisted configuration values in session config: spark.master
 
 edit `conf/spark-blacklist.conf` file in livy server and comment out `#spark.master` line.
+
+if you choose to work on livy https://github.com/cloudera/hue/tree/master/apps/spark/java,
```

Comment:
Hi @AhyoungRyu 
This version livy is embedded in hue repo, and I will make direct link to hue